docs/crio.conf.5.md

```diff
@@ -25,143 +23,188 @@ No bare options are used. The format of TOML can be simplified to:
+**default_mounts_file**="/etc/containers/mounts.conf"
+  Path to the file specifying the defaults mounts for each container. The format of the config is /SRC:/DST, one mount per line. Notice that CRI-O reads its default mounts from the following two files:
 
-**apparmor_profile**=""
-  Name of the apparmor profile to be used as the runtime's default (default: "crio-default")
+    1) `/etc/containers/mounts.conf` (i.e., default_mounts_file): This is the override file, where users can either add in their own default mounts, or override the default mounts shipped with the package.
 
-**no_pivot**=*true*|*false*
-  Instructs the runtime to not use pivot_root, but instead use MS_MOVE
+    2) `/usr/share/containers/mounts.conf`: This is the default file read for mounts. If you want CRI-O to read from a different, specific mounts file, you can change the default_mounts_file path right below. Note, if this is done, CRI-O will only add mounts it finds in this file.
 
-**default_mounts**=[]
-  List of mount points, in the form host:container, to be mounted in every container
+**pids_limit**=1024
+  Maximum number of processes allowed in a container.
+
+**log_size_max**=-1
+  Maximum sized allowed for the container log file. Negative numbers indicate that no size limit is imposed. If it is positive, it must be >= 8192 to match/exceed conmon's read buffer. The file is truncated and re-opened so the limit is never exceeded.
+
+**container_exits_dir**="/var/run/crio/exits"
+  Path to directory in which container exit files are written to by conmon.
+
+**container_attach_socket_dir**="/var/run/crio"
+  Path to directory for container attach sockets.
+
+**read_only**=false
+  If set to true, all containers will run in read-only mode.
+
+**log_level**="error"
+  Changes the verbosity of the logs based on the level it is set to. Options are fatal, panic, error, warn, info, and debug.
+
+**uid_mappings**=""
+  The UID mappings for the user namespace of each container. A range is specified in the form containerUID:HostUID:Size. Multiple ranges must be separated by comma.
+
+**gid_mappings**=""
+  The GID mappings for the user namespace of each container. A range is specified in the form containerGID:HostGID:Size. Multiple ranges must be separated by comma.
+
+**ctr_stop_timeout**=10
+  The minimal amount of time in seconds to wait before issuing a timeout regarding the proper termination of the container.
 
-**read_only**==*true*|*false*
-  Run every container in read-only mode. Automatically mount tmpfs on `/run`, `/tmp` and `/var/tmp`.
-  Setup images to run in read-only. (default: false)
 
 ## CRIO.IMAGE TABLE
+The `crio.image` table contains settings pertaining to the management of OCI images.
 
-**default_transport**
-  A prefix to prepend to image names that can't be pulled as-is (default: "docker://")
+**default_transport**="docker://"
+  A prefix we for images that cannot be parsed as a valid source reference.
```

Comment:
not sure what you're trying to say in this sentence.  Perhaps drop "we"?

Reply:
`Default transport for pulling images from remote container storage.`